src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1504,7 +1438,7 @@ inline RppStatus rpp_store4_f32pln3_to_u8pkd3(Rpp8u* dstPtr, __m128* p)
     __m128 p1 = _mm_unpacklo_ps(p[0], p[1]);
```

Comment:
I think the naming convention is changing here a little. Lets follow the same convention like elsewhere. Isn't this function the same as the rpp_store12_f32pln3_to_f32pkd3() function above, except it stores in U8. So it should ideally be called rpp_store12_f32pln3_to_u8pkd3().